docs/cheqd-cli/cheqd-cli-identity-tutorial.md

```diff
@@ -0,0 +1,413 @@
+      "type": "Ed25519VerificationKey2020",
+      "controller": "did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe",
+      "public_key_multibase": "z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe3Nmod35uua9TE"
+    }
+  ],
+  "authentication": [
+    "did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe#key1"
+  ]
+}
+```
+
+  
+
+We recommend you store this DIDDoc in a separate file, like `json.txt` and inject it while running the command for sending.
+
+  
+
+### 5\. Send DIDDoc to the pool
+
+  
+
+Now that we have our DIDDoc prepared we can send it to the pool.
+
+  
+
+We can use the following command to send the DIDDoc:
+
+  
+
+```
+$ cheqd-noded tx cheqd create-did "$(cat json.txt)" \
+"did:cheqd:testnet:zJ5EDiiiKWDyo79n#key1" \
+--ver-key "FxaJOy4HFoC2Enu1SizKtU0L+hmBRBAEpC+B4TopfQoyetOF5T68Ks3db5Yy9ykFdgEboPUes3m6wvXNLpbv+Q==" \
+  --from  --node https://rpc.testnet.cheqd.network:443 --chain-id cheqd-testnet-4 --fees 5000000ncheq
+```
+
+  
+
+Where:
+
+*   `"did:cheqd:testnet:zJ5EDiiiKWDyo79n#key1"` is the `id` of `verification_method` section
+*   `--ver-key` - is from `keys.txt` file from the [step](#1-generate-verification-key), `priv_key_base_64` field.
+*   Instead of `--fees` option, the `--gas-prices "25ncheq"` can be used also.
+*   `--from` - should be an alias of your cosmos keys.
+
+  
+
+After you execute the command you will receive `"code": 0"`if the DID was successfully written to the ledger. We can do a full query to check this as well.
+
+  
+
+### 6\. Check that DID was successfully written to the ledger
+
+  
+
+Finally, to check that the DID was successfully written we can use the following query:
+
+  
+
+```
+$ cheqd-noded query cheqd did "<identifier-of-your-DIDDoc>" --node https://rpc.testnet.cheqd.network:443
+```
+
+  
+
+where:
+
+*   `<identifier-of-your-DIDDoc>` - identifier with template `"did:cheqd:<namespace>:<unique-id>"` and `<unique-id>` is from [step](#3-create-unique-id-for-our-did)
+
+  
+
+In our example:
+
+```
+$ cheqd-noded query cheqd did "did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe" --node https://rpc.testnet.cheqd.network:443
+```
+
+  
+
+You can also check this using an API. The API path for requesting a DIDDoc on testnet is:
+
+```
+https://api.testnet.cheqd.network/cheqd/cheqdnode/cheqd/did/<identifier-of-your-DIDDoc>
+```
+
+  
+
+**Congratulations! You've created your first, of many, DIDDoc on cheqd!**
+
+  
+
+* * *
+
+Requirements from OS side
+-------------------------
+
+  
+
+Our target OS system is Ubuntu 20.04.
+
+  
+
+In this case, for running demo flow we can use variants: Virtualbox or docker.
+
+For example, let it be a docker image, cause it's the most fastest way to start playing.
+
+The next command can help:
+
+  
+
+```
+$ docker run -it --rm -u root --entrypoint bash ghcr.io/cheqd/cheqd-node:0.4.0
+```
+
+  
+
+After that, we need to install needed package for process SSL certificates:
+
+  
+
+```
+# apt update && apt install ca-certificates -y
```

Comment:
Yes, but I used our official image 0.4.0 version without ca-certificates inside. If we choose to use the latest one then it can be removed of course.